portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,7 +13,8 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
+import com.google.gson.Gson;
```

Comment:
You could refer to https://google.github.io/styleguide/javaguide.html for more useful style guide for writing Java code. I think it is generally the practice to sort the imports in alphabetical order.

Reply:
Thank you for this!